src/main/webapp/app/course/manage/course-update.component.scss

```diff
@@ -34,10 +42,27 @@
     }
 }
 
-image-cropper {
-    padding: 0 0 10px 0 !important;
+.icon-button-wrapper {
+    position: absolute;
+    background-color: var(--gray-500);
```

Comment:
Just a suggestion: the gray background color combined with the colored outline struck me as a bit unusual considering the design of buttons in Artemis in general. Could you look into the color scheme of other buttons and see how it would look if you applied them here?

https://docs.artemis.cit.tum.de/dev/guidelines/client-design.html describes a few ways you can do this, e.g. by using classes for the button (warning / danger would be the ones to look for here).

Reply:
While creating the buttons I was referring to the [Figma design](https://www.figma.com/file/dFkgG08cTwMVYdkialksWH/Course-Avatar?type=design&node-id=0%3A1&mode=design&t=ykjvW0j7kOSKlHDM-1) that @bassner created

Some other possible designs:
(bs-body-bg in the background)

Delete button -> danger, Edit button -> body-color
<img width="200" alt="Screenshot 2024-04-17 at 23 10 33" src="https://github.com/ls1intum/Artemis/assets/23586149/449d3c7c-b90d-4d84-ab20-fe9c8a079d7b">
<img width="200" alt="Screenshot 2024-04-17 at 23 10 47" src="https://github.com/ls1intum/Artemis/assets/23586149/b6d3e930-cdc6-4d4f-9a9e-7b99e6e9458a">

Delete button -> danger, Edit button -> primary
<img width="200" alt="Screenshot 2024-04-17 at 23 12 44" src="https://github.com/ls1intum/Artemis/assets/23586149/46d8059b-56b9-459e-9695-20534938754b">
<img width="200" alt="Screenshot 2024-04-17 at 23 12 52" src="https://github.com/ls1intum/Artemis/assets/23586149/e5edb249-9295-4c4b-9dc5-d752fcf76fe5">

Delete button -> danger, Edit button -> secondary
<img width="200" alt="Screenshot 2024-04-17 at 23 13 26" src="https://github.com/ls1intum/Artemis/assets/23586149/455843c7-fb77-486a-ba4a-c388f45b2dd5">
<img width="200" alt="Screenshot 2024-04-17 at 23 13 36" src="https://github.com/ls1intum/Artemis/assets/23586149/b01f3bec-56ba-48cb-a3b7-90435dde9ec5">

Please let me know which one would be better

Reply:
I'd say we can go with secondary 👍

Reply:
Changed